projects/fal/src/fal/cli/teams.py

```diff
@@ -76,8 +41,8 @@ def add_parser(main_subparsers, parents):
         help=set_help,
         parents=parents,
     )
-    set_parser.add_argument("team", help="The team to set.")
-    set_parser.set_defaults(func=_set)
+    set_parser.add_argument("account", help="The team to set.")
```

Comment:
not sure why this was needed. you shouldn't need to set personal account, you have `unset` for that.